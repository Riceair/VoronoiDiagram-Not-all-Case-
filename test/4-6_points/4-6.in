4
200 200
300 200
400 200
500 200