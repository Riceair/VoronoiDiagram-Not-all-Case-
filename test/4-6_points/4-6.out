P 200 200
P 300 200
P 400 200
P 500 200
E 350 -62800 350 63200
